components/interactive_dialog/dialog_element/dialog_element.tsx

```diff
@@ -177,7 +177,7 @@ export default class DialogElement extends React.PureComponent<Props, State> {
                     placeholder={placeholder}
                     helpText={helpTextContent}
                     onChange={onChange}
-                    resizable={false}
+                    resizable={true}
```

Comment:
We still need to make sure this is only applied for textareas. This can be done like so:

```suggestion
                    resizable={type === 'textarea'}
```